PANDORA/Pandora/Modelling_functions.py

```diff
@@ -131,7 +131,7 @@ def predict_anchors_netMHCIIpan(peptide, allele_type, output_dir, verbose=True,
     # Retrieves the enviroment variable netMHCIIpan
     netmhcpan_file_path = set([x for x in [os.getenv('netMHCIIpan', default=None), 
                           os.popen('which netMHCIIpan').read().strip()] 
-                          if type(x) == str and re.search("\/netMHCIIpan\-\d+\.\d+\/netMHCIIpan$", x)])
+                          if type(x) == str])
```

Comment:
Can I ask you why did you remove this strict check? I agree is a bit redundant, but was it causing any specific issue?

Reply:
I have a working version of netMHCpan installed on windows in wsl. I have it placed in a folder under a custom name so other users on the workstation don't confuse it.
netMHCpan can be difficult and inflexible to install with the way its implemented using absolute paths. So I was just happy to have it running. But then pandora just threw an error stating it's not installed even though it's perfectly usable when removing the check.
I am not sure what os.getenv('netMHCIIpan') does but if "which netMHCpan" works I am not sure it's wise to add further restrictions. Maybe one do a quick check of netMHCpan --version or something like this.
I mentioned in the commit message: "If this is unavoidable I would request for this requirement to be documented or a precise error to be thrown". Otherwise it's likely to cause a lot of confusion finding out what is the point of failure.

Reply:
I see, thanks a lot for the explanation.
The reason why we used both os.getenv and netmhcpan is because, depending on how the user installed netmhcpan (i.e. by setting an alias or by adding it to the PATH) they return different results, so we wanted to make sure to catch them all.